cpp/CMakeLists.txt

```diff
@@ -120,7 +120,7 @@ if(APPLE)
                       OUTPUT_STRIP_TRAILING_WHITESPACE)
     endif()
 
-    execute_process(COMMAND ${BREW_BIN} --prefix "llvm@${ARROW_CLANG_TOOLS_MAJOR_VERSION}"
+    execute_process(COMMAND ${BREW_BIN} --prefix "llvm@${ARROW_CLANG_TOOLS_VERSION_MAJOR}"
```

Comment:
This is not related to CI fix.
I just find this. So I put this tiny fix this pull request. Sorry.